components/Mitra/index.vue

```diff
@@ -336,6 +337,48 @@ export default {
     },
     addNewMitra () {
       this.showModalAddMitra = true
+    },
+    refreshDatatable () {
+      this.query = {
+        per_page: 5,
+        sort_by: 'desc',
+        order_by: 'users.created_at',
+        current_page: 1,
+        is_admin: false,
+        roles: 'mitra'
+      }
+      this.$fetch()
+    },
+    async actiondeleteDataPartner (item) {
```

Comment:
minor:

Hello @doohanas , if we're using `camelCase` naming convention, this method should be named` actionDeleteDataPartner`

ref: https://wiki.digitalservice.id/doc/js-naming-conventions-gRnYoPmhnd

Reply:
hello @Ibwedagama, You're right I miss naming this. I will refactor it soon. Thanks